tox.ini

```diff
@@ -1,23 +1,22 @@
 [tox]
 envlist =
-    py{36,37,38,39}-dj{22,30,31}
+    py{37,38,39}-dj{22,31,32}
     isort
     docs
 whitelist_externals=sphinx-build
 
 [gh-actions]
 python =
-    3.6: py36
     3.7: py37
     3.8: py38
     3.9: py39,isort,docs
 
 [testenv]
 deps=
     -rrequirements-dev.txt
-    dj22: django>=2.2a1,<3.0
-    dj30: django>=3.0a1,<3.1
-    dj31: django>=3.1a1,<3.2
+    dj22: Django==2.2
+    dj31: Django>=3.1,<3.2
+    dj32: Django>=3.2,<3.3
```

Comment:
You could use the new `~=` version syntax if you want the latest bugfix release of a minor version.